charts/shopware/values.yaml

```diff
@@ -31,8 +31,8 @@ percona:
     # annotations:
     # affinity:
     # serviceAnnotations:
-    # storageClassName:
-    # storageSize: 8Gi
+    # storageClassName: standard
+    storageSize: 8Gi
```

Comment:
Change the default in the helm chart and comment it here, as we do with all other values too